README.md

```diff
@@ -450,14 +451,13 @@ ctest -j8
 * **version control** (optional): git.
     * Xcode Command Line Tools gives you git on the command line.
     * [GitHub for Mac](https://mac.github.com), for a simple-to-use GUI.
-* **Bindings** (optional): [SWIG](http://www.swig.org/) 3.0.6
+* **Bindings** (optional): [SWIG](http://www.swig.org/) 3.0.8
     * **MATLAB scripting** (optional): [Java development kit][java] >= 1.7.
         * Note: Older versions of MATLAB may use an older version of JVM. Run
                 'ver' in MATLAB to check MATLAB's JVM version (must be >= 1.7).
-    * **Python scripting** (optional):
+    * **Python scripting** (optional): Python >= 2.7 (including Python 3)
```

Comment:
Good point. I tested with Python 3.5.1. I've edited the README to specify `>= 3.5`. What do you think?